### PR TITLE
Include python 3.8 support on all platforms

### PR DESCRIPTION
**Pull request by ernestoarbitrio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,6 +28,17 @@ jobs:
         - brew install openslide
         - echo 'export PATH="/usr/local/Frameworks/Python.framework/Versions/Current/bin:$PATH"' >> /Users/travis/.bash_profile
         - echo $PATH
+    - name: "Python: 3.8"
```
Comment (alessiamarcolini) [inline .travis.yml:31]: here it's running with Python 3.7.5

Comment (alessiamarcolini) [reply .travis.yml:31]: you need to specify the osx image:

I know that `xcode10.2` comes with py 3.7.3 (https://blog.travis-ci.com/2019-08-07-extensive-python-testing-on-travis-ci)

So I guess we need to find the right one between `xcode10.3` and `xcode12u`. Complete list https://docs.travis-ci.com/user/reference/osx/

Comment (ernestoarbitrio) [reply .travis.yml:31]: Why we need to specify the osx image if we have the proof that it's running with the right python version?

Comment (ernestoarbitrio) [reply .travis.yml:31]: mmmm maybe u right

Comment (alessiamarcolini) [reply .travis.yml:31]: you can see the actual python version by looking at the pytest log on travis

Comment (ernestoarbitrio) [reply .travis.yml]: yes i see

File: .travis.yml [hunk 2]
```diff
@@ -26,6 +28,14 @@ jobs:
         - brew install openslide
         - echo 'export PATH="/usr/local/Frameworks/Python.framework/Versions/Current/bin:$PATH"' >> /Users/travis/.bash_profile
         - echo $PATH
+    - name: "Python 3.8.5 on macOS"
```
Comment (alessiamarcolini) [inline .travis.yml]:
```suggestion
    - name: "Python 3.8.3 on macOS"
```

File: .travis.yml [hunk 3]
```diff
@@ -44,6 +54,15 @@ jobs:
         - wget https://github.com/openslide/openslide-winbuild/releases/download/v20171122/openslide-win64-20171122.zip -P /c/downloads
         - 7z e /c/downloads/openslide-win64-20171122.zip -aoa
       env: PATH=/c/Python37:/c/Python37/Scripts:/c/downloads/openslide-win64-20171122/bin:$PATH
+    - name: "Python 3.8 on Windows"
```
Comment (alessiamarcolini) [inline .travis.yml:57]:
```suggestion
    - name: "Python 3.8.0 on Windows"
```

**Review by alessiamarcolini — APPROVED** [review]

**Pull request merged**